composer.json

```diff
@@ -0,0 +1,63 @@
+{
+    "name": "keboola/minimalist-cors",
```

Comment:
a nie proste len "keboola/cors" ?

Reply:
jojo, to byla původní verze pak jsem to přejmenoval a tady zapomněl!